omnibus/files/private-chef-scripts/gather-logs

```diff
@@ -21,6 +21,14 @@ then
     testing=true
 fi
 
+#Don't continue if bzip isn't installed
```

Comment:
Do we not ship with bzip? Should bzip be a dependency in chef server omnibus/hab packages? Or, alternatively is there something (gzip) that we do ship? What's the performance cost of switching?

I'd love to figure out how to have a 'batteries included' system so this check isn't needed.

Reply:
I'm guessing that compression (to meet zendesk's 20MB/file limit) was the reason to use bzip, and bzip2 is not a hard dependency of the package (whether debian or EL) right now.

Reply:
switching to gzip and removing check